README.md

```diff
@@ -1,24 +1,34 @@
 # GISMap
 GIS map being developed with AFS (initially for Parking Services)
 
-[App homepage on GIS Confluence](https://esdconfluence.it.umass.edu/confluence/display/AFGIS/GIS+-+App+Development+-+Campus+Parking+Map)
+[App homepage on GIS Confluence][wiki-gis]
 
-[Confluence backend documentation](https://esdconfluence.it.umass.edu/confluence/display/AFGIS/GIS+-+Parking+App+-+Backend)
+[Confluence backend documentation][wiki-backend]
 
 ## Setup
 [Install Yarn](https://yarnpkg.com/en/docs/install)
 
-Install compilation dependencies with:
+Install dependencies with:
+
 ```
-yarn global add typescript
-yarn global add browserify
+$ yarn
 ```
 
 Compile the application with:
+
 ```
-./compile.sh
+$ yarn run build
+```
+
+The built site will be in the `out/` directory. Serve it with:
```

Comment:
What do you mean by "serve it"?

Reply:
Start a webserver? I can be a bit more explicit, I guess. Although, I think the output is (moderately) explanatory:

```
$yarn run serve
yarn run v1.16.0
$ yarn run static out/
$ /home/matt/projects/GISMap/node_modules/.bin/static out/
serving "out/" at http://127.0.0.1:8080
```